.lintstagedrc

```diff
@@ -5,7 +5,7 @@
 	"*.ts": [
 		"balena-lint --typescript --fix",
 	],
-	"*!(*webpack.config).js": [
+	"*.js": [
```

Comment:
NIT: You can have typescript webpack config files

Reply:
That's cool, we're not specifically targeting it here though, we just aren't not-targeting it :P